_summaries/wang2018dynamic_2.md

```diff
@@ -1,19 +1,41 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {wang2018Ddynamics}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* What is the core idea? <br/>
+In this paper, the authors come up with a new neural network module called EdgeConv that can use local geometric information of point clouds, which is permutational invariant and translational invariant. By dynamically updating a graph of relationships, EdgeConv is able to perform semantic segmentation task. EdgeConv can also easily be integrated into other point cloud processing pipelines. The following picture shows the entire model architecture for classification and segmentation,<br/>
+<img src = 'model_arch.png'>
+
+
+* How is it realized (technically)? <br/>
```

Comment:
Before jumping into the expressions below, maybe the functions/variables can be defined first.   Unless you've already read the paper, I don't think the following diagram and equations really make sense.  You could also include Table 1 from the paper for further clarity.

Reply:
Thanks for letting me know this problem. I think I probably just remain the written summary in this way, but I will mention functions/variables of EdgeConv first when I do my presentation.